build/configure

```diff
@@ -90,6 +90,11 @@ case $uname in
 	fi
 
         case `uname -m` in
+	    # ARCH is only used for OMS to determine the correct ruby folder
```

Comment:
Really? You sure on that? If so, that must have been a real joy to figure out! ☹️